docs/tutorial/code/mygc_semispace/global.rs

```diff
@@ -201,8 +199,26 @@ impl<VM: VMBinding> MyGC<VM> {
                 mmapper,
                 &mut heap,
             ),
-            common: CommonPlan::new(vm_map, mmapper, options, heap, &MYGC_CONSTRAINTS, global_metadata_specs),
-        }
+            common: CommonPlan::new(vm_map, mmapper, options, heap, &MYGC_CONSTRAINTS, global_metadata_specs.clone()),
+        };
+
+        let mut side_metadata_sanity_checker = SideMetadataSanity::new();
+        side_metadata_sanity_checker.verify_metadata_context(
+            "CopySpace",
+            &SideMetadataContext {
+                global: global_metadata_specs.clone(),
+                local: Vec::from(res.copyspace0.local_side_metadata_specs()),
+            },
+        );
+        side_metadata_sanity_checker.verify_metadata_context(
+            "CopySpace",
```

Comment:
Is this correct? According to the code in `verify_metadata_context()`, if we have the same `policy_name`, it is not `first_call`, it will panic if the array already contains the local spec.

Reply:
It will panic if the new list of local metadata is not the same as the first one. (e.g. if any of the `spec`s in the new list is not contained in the list from first call)